NAS2D/Resource/AnimationSet.cpp

```diff
@@ -50,7 +50,7 @@ bool AnimationSet::Frame::isStopFrame() const
 }
 
 
-AnimationSet::AnimationSet(std::string fileName) :
+AnimationSet::AnimationSet(const std::string& fileName) :
 	mFileName{fileName}
 {
```

Comment:
I think the more proper solution here is to use `std::move`.

```cpp
mFileName{std::move(fileName)}
```

You don't want to be passing by `const&` when using `std::move`. That would end up silently forcing a copy instead.